### PR TITLE
WIP: tools: added test for prplmesh operational status

### PR DESCRIPTION
**Pull request by slyusar003**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,9 @@ services:
 script:
   - cd tools/docker
   - sudo ./image-build.sh -b ubuntu:18.04
-  - sudo ./build.sh map -f MSGLIB=zmq BUILD_TESTS=ON
+  - sudo ./build.sh build map -f MSGLIB=zmq BUILD_TESTS=ON
+  - sudo ./test.sh test -c start --skip-platform
```
Comment (tomereli) [inline .travis.yml]:
```suggestion
sudo ./run.sh start-controller-agent -d -n cont
sudo docker container exec -it cont <path to install>/scripts/prplmesh_utils.sh status
```

File: common/CMakeLists.txt [hunk 2]
```diff
@@ -14,9 +14,14 @@ include(GNUInstallDirs)
 
 if (TARGET_PLATFORM STREQUAL "ugw")
     add_definitions(-DBEEROCKS_UGW)
+    set(BEEROCKS_TMP_PATH "/tmp/beerocks")
 elseif (TARGET_PLATFORM STREQUAL "rdkb")
     add_definitions(-DBEEROCKS_RDKB)
     set(CMAKE_SKIP_RPATH TRUE)
+    set(BEEROCKS_TMP_PATH "/tmp/beerocks")
+elseif (TARGET_PLATFORM STREQUAL "linux")
+    add_definitions(-DBEEROCKS_LINUX)
```
Comment (tomereli) [inline common/CMakeLists.txt]: Why do you need to add CFLAG here? `BEEROCKS_LINUX`

File: tools/docker/builder/Dockerfile [hunk 3]
```diff
@@ -21,6 +21,12 @@ RUN apt-get update && apt-get install -y \
 	libzmq3-dev \
 	python \
 	python-yaml \
+	net-tools \
```
Comment (tomereli) [inline tools/docker/builder/Dockerfile]: This is a builder dockerfile - please don't change it, you should use the runner dockerfile for testing

File: tools/docker/README.md [hunk 4]
```diff
@@ -17,12 +17,12 @@ If `build.sh` or `run.sh` are called without first calling `image-build.sh`, the
 
 ## Docker builder
 
-The docker builder image can be used to build prplMesh via `maptools.py build` command running in a prplmesh-build container via `build.sh`.
```
Comment (tomereli) [inline tools/docker/README.md]: builder docker should remain only for build. You have runner docker for anything else.

File: tools/docker/test.sh [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+run() {
+    echo "$*"
+    "$@" || exit $?
+}
+
+scriptdir="$(cd "${0%/*}"; pwd)"
+topdir="${scriptdir%/*/*/*/*}"
+
+main() {
+    run docker container run -u root -v ${topdir}:${topdir} --interactive --tty --rm prplmesh-build $@
```
Comment (tomereli) [inline tools/docker/test.sh:12]: simply use prplmesh-runner instead here, but also base this script on run.sh which added some more capabilities (maybe you need to rebase)

**Review by tomereli — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 6]
```diff
@@ -10,6 +10,8 @@ script:
   - cd tools/docker
   - sudo ./image-build.sh -b ubuntu:18.04
   - sudo ./build.sh map -f MSGLIB=zmq BUILD_TESTS=ON
+  - sudo ./run.sh start-controller-agent -d -n cont
+  # - sudo ./test.sh test -c status
```
Comment (tomereli) [inline .travis.yml:14]: remove commented out call

File: tools/commands/test.py [hunk 7]
```diff
@@ -0,0 +1,47 @@
+import logging
```
Comment (tomereli) [inline tools/commands/test.py:1]: Just out of curiosity, why do we need this command at all now that prplmesh_utils.sh provides all this?

File: tools/docker/test.sh [hunk 8]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```
Comment (tomereli) [inline tools/docker/test.sh:1]: This script is not needed any more

**Review by tomereli — CHANGES_REQUESTED** [review]

**Comment by arnout** [comment]
This PR has been superseded by #92

**Pull request closed without merging**